src/lib/Event/Listener/LoginListener.php

```diff
@@ -100,21 +101,36 @@ public function onSecurityInteractiveLogin(InteractiveLoginEvent $event): void
         }
     }
 
-    /**
-     * Returns notification API end-point.
-     */
-    private function getNotificationEndpoint(): string
+    private function getCustomerId(string $siteAccessName): ?int
     {
-        $customerId = $this->configResolver->getParameter(
+        return $this->configResolver->getParameter(
             'authentication.customer_id',
-            Parameters::NAMESPACE
+            Parameters::NAMESPACE,
+            $siteAccessName
         );
-        $trackingEndPoint = $this->configResolver->getParameter(
+    }
+
+    private function getEndpoint(string $siteAccessName): ?string
+    {
+        return $this->configResolver->getParameter(
             Parameters::API_SCOPE . '.event_tracking.endpoint',
-            Parameters::NAMESPACE
+            Parameters::NAMESPACE,
+            $siteAccessName
         );
+    }
 
-        return sprintf('%s/api/%s/', $trackingEndPoint, $customerId);
+    /**
+     * Returns notification API end-point.
+     */
+    private function getNotificationUri(string $endpoint, int $customerId, InteractiveLoginEvent $event): string
+    {
+        return sprintf('%s/api/%d/%s/%s/%s',
+            $endpoint,
+            $customerId,
+            'login',
+            $event->getRequest()->cookies->get(RecommendationSession::RECOMMENDATION_SESSION_KEY),
+            $this->getUser($event->getAuthenticationToken())
+        );
     }
```

Comment:
This method should look like: 

```suggestion
    private function getNotificationUri(string $endpoint, int $customerId, string $sessionId, string $userId): string
    {
        return sprintf('%s/api/%d/%s/%s/%s',
            $endpoint,
            $customerId,
            'login',
            $sessionId,
            $userId
        );
    }
```